### PR TITLE
Common test cases

### PR DESCRIPTION
**Pull request by MariuszSkamra**
This PR is contains initial work to get rid of duplicated 'ptsprojects' from autopts. The aim is to run test cases enabled in PTS workspace. The autopts client side will only handle wid and initial conditions.

### CODE REVIEW

**Comment by MariuszSkamra** [comment]
@michal-narajowski FYI

**Comment by tedd-an** [comment]
Test run failed with 3rd and 4th iutconfig. 
The test cases are empty.
```
[182/187] Linking C executable zephyr/zephyr_prebuilt.elf
Memory region         Used Size  Region Size  %age Used
           FLASH:      186669 B         1 MB     17.80%
            SRAM:       38840 B       256 KB     14.82%
        IDT_LIST:         152 B         2 KB      7.42%
[187/187] Linking C executable zephyr/zephyr.elf
[====================] 100%
0017263:INFO:loader:Programmed 186669 bytes (46 pages) at 11.05 kB/s (0 pages unchanged)

Summary:

Status   Count
==============
==============
Total        0
```
[pr_327_error_log.txt](https://github.com/intel/auto-pts/files/3374329/pr_327_error_log.txt)

**Comment by MariuszSkamra** [comment]
@tedd-an I'll try to fix this

**Comment by MariuszSkamra** [comment]
@tedd-an done

File: bot/zephyr.py [hunk 1]
```diff
@@ -334,13 +336,12 @@ def run_tests(args, iut_config):
         results.update(results_dict)
         autoprojects.iutctl.cleanup()
 
-    for test_case_name in results.keys():
+    for project_name, test_case_name in results.keys():
```
Comment (tedd-an) [inline bot/zephyr.py]: project_name is invalid.

```
Traceback (most recent call last):
  File "./autoptsclient_bot.py", line 63, in <module>
    main()
  File "./autoptsclient_bot.py", line 54, in main
    project2main[project['name']](project)
  File "/home/tester/working/auto-pts/bot/zephyr.py", line 410, in main
    run_tests(args, cfg.get('iut_config', {}))
  File "/home/tester/working/auto-pts/bot/zephyr.py", line 339, in run_tests
    for project_name, test_case_name in results.keys():
ValueError: too many values to unpack
```

Comment (tedd-an) [reply bot/zephyr.py]: Fixed it on the fly

**Pull request merged**